core/app/models/spree/promotion/rules/item_total.rb

```diff
@@ -6,20 +6,43 @@ module Rules
       # A rule to apply to an order greater than (or greater than or equal to)
       # a specific amount
       class ItemTotal < PromotionRule
+        include ActiveSupport::Deprecation::DeprecatedConstantAccessor
+
         preference :amount, :decimal, default: 100.00
         preference :currency, :string, default: ->{ Spree::Config[:currency] }
-        preference :operator, :string, default: '>'
+        preference :operator, :string, default: 'gt'
```

Comment:
What about existing rules that are using the old operator syntax (`>`, `>=`) instead of `gt`, `gte`?

Reply:
Oh right, now I recall it! I think a separate commit would be enough.

Reply:
✅ Done.